### PR TITLE
Automatically confirm funding if playing auto opponent.

### PR DESCRIPTION
**Pull request by andrewgordstewart**

### CODE REVIEW

**Comment by tomclose** [comment]
:( that's my fault ...

File: src/constants.ts [hunk 1]
```diff
@@ -4,3 +4,6 @@ export const BRAND_COLOR = style.brandColor;
 
 export const CHALLENGE_REFRESH_INTERVAL = 4000; // milliseconds
 export const CHALLENGE_EXPIRATION_INTERVAL = 5000; // milliseconds
+
+export const AUTOOPPONENT_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000f00'
```
Comment (tomclose) [inline src/constants.ts]: `AUTO_OPPONENT_PRIVATE_KEY` ?

File: src/redux/game/saga.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { default as positionFromHex } from '../../game-engine/positions/decode';
 import { PlayerAStateType } from '../../game-engine/application-states/PlayerA';
 import { PlayerBStateType } from '../../game-engine/application-states/PlayerB';
 
-export default function* gameSaga(gameEngine: GameEngine) {
+export default function* gameSaga(gameEngine: GameEngine, playingAutoOpponent=false) {
```
Comment (tomclose) [inline src/redux/game/saga.ts]: I think we can remove this?

File: src/redux/game/saga.ts [hunk 3]
```diff
@@ -32,6 +32,9 @@ export default function* gameSaga(gameEngine: GameEngine) {
 
     switch (action.type) {
       case messageActions.MESSAGE_RECEIVED:
+        if (playingAutoOpponent) {
+          gameEngine.fundingConfirmed()
+        }
```
Comment (tomclose) [inline src/redux/game/saga.ts]: And this?

File: src/redux/game/saga.ts (hunk 2 above)
Comment (andrewgordstewart) [reply src/redux/game/saga.ts]: Not sure how this stayed in :/

**Pull request merged**